### DISCUSSION TITLE
Confusion with columns

### DISCUSSION THREAD

**Discussion by CatholicPenguin** [sile-typesetter/sile#1790] [Q&A]
In the book class, I have tried having parallel columns followed by single column text, but I don't see in the manual how to switch between them.

Also, using the bible class (but also true in the book class), I don't know how to have text automatically put into two columns throughout the entire document. Though, I would need a single column section for the title, for example.

Could someone help with that? I'm just getting used to looking at lua files.

**Comment by Omikhleia** [comment]
Switching to a two-column page template is described in the manual (see "Frames" in chapter "The Nitty Gritty").

For parallel texts (something slightly different, for synchronized columns -- but I'm not sure which case you were considering), there's also the diglot class, with general concepts explained in the manual, though it probably remains experimental, I daresay.

As for having a two-column layout but sometimes go back to single column on a page (for section title etc.), there's nothing yet out of the box (though the frametricks package may help there, maybe? It may call for more troubles, though!)

All in all, I am not sure there's a good workflow yet (generic and sustainable) for this kind of presentation, though I'd be glad to be proven wrong ;)
